src/Akeneo/Bundle/ElasticsearchBundle/Refresh.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Akeneo\Bundle\ElasticsearchBundle;
+
+/**
+ * @author    Arnaud Langlade <arnaud.langlade@@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+final class Refresh
+{
+    const ENABLE = true;
+    const DISABLE = false;
+    const WAIT_FOR = 'wait_for';
+
+    /** @var bool|string */
+    private $type;
+
+     function __construct($type)
+    {
+        $this->type = $type;
+    }
+
+    /**
+     * @return Refresh
+     */
+    public static function enableParam()
```

Comment:
I would remove the "param" suffix to all the methods personnally
I don't understand the meaning